CRUDRecipeEF.BL.DL/Services/RecipeService.cs

```diff
@@ -78,6 +78,19 @@ public async Task<string> AddIngredientToRecipe(IngredientAddDTO ingredientAddDT
             return recipeName;
         }
 
+        public async Task UpdateRecipe(string name, RecipeAddDTO recipeAddDTO)
```

Comment:
I recommend doing it the way we did with automapper in the ingredientservice. Msg me on discord if you want me to explain if it isnt clear whats happening